### DISCUSSION TITLE
Change Windows user without disconnection

### DISCUSSION THREAD

**Discussion by Freewalkr** [LizardByte/community#273] [Sunshine] [answered]
I'm not sure if this should be asked here or in Moonlight repo, but it seems to be server functionality, so here we go.

Me and my brother use the same VM with two Windows users to play games via Sunshine+Moonlight setup. When I want to use the VM after my brother, I connect to VM via Moonlight (so that I see brother's desktop), change user to mine from Start menu and... screen becomes black and never responds. After disconnection, if I try to connect again, Moonlight says "Connection terminated" after a couple of seconds of trying to resume the session. It's like Sunshine can't display a login screen.

I solve this problem by connecting to the VM via NoMachine. When I connect, it shows login screen without any problems. When I enter password, VM becomes available from Moonlight again. It's not too hard to do it every time, but it's kinda uncomfortable.

Is there a possibility to change Windows users without using other remote desktop services? Maybe I should configure my setup in some way? Maybe I should open a feature request issue?

**Comment by ReenigneArcher** [comment, marked as answer]
Switching users mid-stream is not supported.

**Reply by jfly** [reply]
@ReenigneArcher, can you share any additional information on this? What would be involved in adding support for this? (I'd be interested to give it a shot if you could point me in the right direction)

[a comment by an automated account is not shown]

**Reply by jfly** [reply]
Still relevant...
